### DISCUSSION TITLE
Bundling multiple directories/files

### DISCUSSION THREAD

**Discussion by ashwarren** [parcel-bundler/parcel#4672] [General]
I may be thinking of this incorrectly (I've only been using ParcelJS for a few days), but this is what I'm trying to do:

**Project structure:**

- index.html
- index.js
- styles.css
- project-1/
  - index.html
  - index.js
  - styles.css
- project-2/
  - index.html
  - index.js
  - styles.css

I have been able to get the sub-directories and .html files to transfer over into the dist/ dir on build but cannot access via localhost:1234/project-1/ I have to manually add the index.html to the url.

Also the js and css files are flattened into the main dist directory and do not transfer into "project-1" etc.

Looks like what I'm trying to do might be related to this: https://github.com/parcel-bundler/parcel/issues/4548

Hopefully this isn't confusing thank you for any help you can provide.

**Comment by colindresj** [comment]
I'm having the same issue

**Comment by maiya-22** [comment]
Would you be willing to share your config/ setup that got you this far?  I got stuck before this point, and am not able to get multiple html files to work.
